integration/oneof-unions/oneof.ts

```diff
@@ -115,8 +117,7 @@ export const PleaseChoose = {
   decode(input: Reader | Uint8Array, length?: number): PleaseChoose {
     const reader = input instanceof Reader ? input : new Reader(input);
     let end = length === undefined ? reader.len : reader.pos + length;
-    const message = { ...basePleaseChoose } as PleaseChoose;
-    message.signature = new Uint8Array();
+    const message = createBasePleaseChoose();
```

Comment:
Here the creator function is actually nice to have. If it creates all fields, its pretty simple.

Does it make sense to just rename create**Base**PleaseChoose to create**Empty**PleaseChoose and leave it for the decode implementation?